test/parallel/test-fs-readfile-pipe.js

```diff
@@ -40,13 +39,28 @@ if (process.argv[2] === 'child') {
   return;
 }
 
+const tmpdir = require('../common/tmpdir');
+
+const filename = path.join(tmpdir.path, '/readfile_pipe_test.txt');
+const dataExpected = `xxxxxxxxxx xxxxxxxxxx
+xxxxxxxxxx xxxxxxxxxx
+xxxxxxxxxx xxxxxxxxxx
+xxxxxxxxxx xxxxxxxxxx
+xxxxxxxxxx xxxxxxxxxx`
+tmpdir.refresh();
+fs.writeFileSync(filename, dataExpected);
```

Comment:
Nit: Instead of creating the file in the tmpdir, maybe it would be better to put the file permanently in `test/fixtures` and read it from there?

Reply:
I agree.  I'll make this change.